pkg/plugins/platform/types.go

```diff
@@ -1,5 +1,7 @@
 package platform
 
+import "log"
```

Comment:
Here I thought you always insisted on `	log "github.com/Sirupsen/logrus"
` :)
I might do a PR that just globally switches that out wherever we have log.